lighthouse-core/gather/driver.js

```diff
@@ -47,8 +47,8 @@ class Driver {
      */
     this._eventEmitter = /** @type {CrdpEventEmitter} */ (new EventEmitter());
     this._connection = connection;
-    // currently only used by WPT where just Page and Network are needed
-    this._devtoolsLog = new DevtoolsLog(/^(Page|Network)\./);
+    // currently used by network-recorder just Page, Network, and Target are needed
```

Comment:
do we need this comment anymore? It's devtoolsLog :)

(at most we could say it's used to provide the artifact of the same name)

Reply:
heh, fair, I made a note that its purpose is to save network traffic. Seems like we need something to denote why we are filtering this way.